src/cascadia/TerminalControl/TermControl.xaml

```diff
@@ -47,7 +47,19 @@
 
                 <SwapChainPanel x:Name="SwapChainPanel"
                                 SizeChanged="_SwapChainSizeChanged"
-                                CompositionScaleChanged="_SwapChainScaleChanged" />
+                                CompositionScaleChanged="_SwapChainScaleChanged">
+
+                    <Canvas x:Name="SubCanvas"
+                            Visibility="Visible">

```

Comment:
If this is visible by default... what is keeping it invisible when no tooltip is shown? Just the fact that it has no content except for the ToolTipService?

Reply:
Turns out that having no content will still show a small tooltip box that's blank. 

But, that does not happen here because there is no border until we actually hover over a link - since the border thickness is initialized as 0 and we only set its size when the pointer touches a cell with a link. So the tooltip is kept 'invisible' in the sense that its impossible to mouse over the border.

Reply:
Ok, so I won't worry about it then.